.gitignore

```diff
@@ -1,2 +1,4 @@
 build
 env
+__pycache__
+symbiflow
```

Comment:
Why is symbiflow directory excluded?

Reply:
Oh, I used a sub-directory to hold the symbiflow toolchain. That usually isn't housed inside symbiflow-examples though, so I'll remove that line.